Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -22,20 +23,41 @@ public class Main : ISettingProvider, IPlugin, IReloadable, IPluginI18n, IContex
         private static List<Bookmark> cachedBookmarks = new List<Bookmark>();
 
         private static Settings _settings;
-        
+
+        private static bool initialized = false;
+
         public void Init(PluginInitContext context)
         {
             Main.context = context;
-            
+
             _settings = context.API.LoadSettingJsonStorage<Settings>();
 
-            cachedBookmarks = BookmarkLoader.LoadAllBookmarks(_settings);
+            LoadBookmarksIfEnabled();
 
-            _ = MonitorRefreshQueue();
+            initialized = true;
```

Comment:
`initialized = true;` should be moved inside `LoadBookmarksIfEnabled` at the end of the method.